### ISSUE TITLE
Command line arguments for segment_julius.pl

### ISSUE THREAD

**Issue by timmahrt** [julius-speech/segmentation-kit#3]
Hello, I am using julius-speech to do forced alignment.
https://github.com/timmahrt/pyJuliusAlign

My code uses an old perl script `segment_julius4.pl` but I would like to use the most recent version.

The most recent one says it takes a command line argument to specify the install location--which my script needs--but it does not.  Can we make the following change?

Line 33 of `segment_julius.pl`:
```
## data directory
$datadir = "./wav";
if (defined $ARGV[0]) {
  $datadir = $ARGV[0];
}
```

**Comment by LeeAkinobu** [comment]
OK, I reproduced the error.  I can't believe myself missing this bug for years...

Fixed on the commit  e0e8bbaf98e27d19dfc6fe8312be607ad03592ad
Thanks for your contribution!

**Comment by timmahrt** [comment]
Thank you for your work in developing and maintaining julius!
